lib/tapioca/commands/dsl.rb

```diff
@@ -222,6 +230,25 @@ def constantize_compilers(compiler_names)
         T.cast(compiler_map.values, T::Array[T.class_of(Tapioca::Dsl::Compiler)])
       end
 
+      sig { params(paths: T::Array[String]).returns(T::Array[String]) }
+      def contants_from_paths(paths)
+        output = T.unsafe(self).sorbet("--no-config", "--quiet", "--print=symbol-table-json", *paths).out
+        constants_from_symbol_table_json(JSON.parse(output))
+      end
+
+      sig { params(symbol_table_json: T::Hash[String, T.untyped], prefix: T.nilable(String)).returns(T::Array[String]) }
+      def constants_from_symbol_table_json(symbol_table_json, prefix = nil)
+        kind = symbol_table_json.fetch("kind")
+        name = symbol_table_json.fetch("name")
+        return [] unless kind == "CLASS_OR_MODULE" && name.fetch("kind") == "CONSTANT"
+
+        name = name.fetch("name")
+        children = symbol_table_json.fetch("children", [])
+
+        prefixed_name = name == "<root>" ? prefix : [prefix, name].compact.join("::")
+        [prefixed_name, *children.flat_map { |c| constants_from_symbol_table_json(c, prefixed_name) }].compact
+      end
+
```

Comment:
I think this logic should be moved to https://github.com/Shopify/tapioca/blob/main/lib/tapioca/static/symbol_loader.rb.

Reply:
Good call. I actually read through that code a bit more and realized it was going exactly what I was trying to do so we can just switch to using it directly. Will push up a commit shortly.